### PR TITLE
properties for keys U83 U84 U85

### PR DESCRIPTION
**Pull request by pasqLisena**
This pull request contains some useful properties for describing the relations between musical keys.
It follows an [issue](https://github.com/DOREMUS-ANR/knowledge-base/issues/38) about the keys vocabulary.

I added the definition and the comments in English and French, so that it is complete for a clear discussion about their adoption.

### CODE REVIEW

File: doremus.ttl [hunk 1]
```diff
@@ -1601,6 +1601,40 @@ Examples:"""@en ;
...
                                   "M9 Type de dérivation"@fr .
 
+###  http://data.doremus.org/ontology#U83_has_relative_key
+mus:U83_has_relative_key rdf:type owl:SymmetricObjectProperty ;
+    rdfs:domain mus:M4_Key ;
+    rdfs:range mus:M4_Key ;
+    rdfs:isDefinedBy <http://data.doremus.org/ontology#> ;
+    rdfs:label "U83 has relative key"@en , "U83 a pour tonalité relative"@fr ;
+    rdfs:comment "Scope Note: This class allows to connect two M4 Keys that are in a relative relationship, i.e. one is the relative major or minor of the other one."@en ,
+    "Scope Note: Cette classe permet de connecter deux M4 Tonalité qui sont en relation par rapport à l'armure, c'est-à-dire l'une est la relatif majeur ou mineur de l'autre."@fr ,
```
Comment (rtroncy) [inline doremus.ttl]: Line 1611: s/l'armure/l'autre

File: doremus.ttl [hunk 2]
```diff
@@ -1601,6 +1601,40 @@ Examples:"""@en ;
...
+    "Exemples: la clé de 'Do majeur' a 'Do mineur' pour tonalité parallèle"@fr .
+
+###  http://data.doremus.org/ontology#U85_has_closely_related_key
+mus:U85_has_closely_related_key rdf:type owl:SymmetricObjectProperty ;
+    rdfs:domain mus:M4_Key ;
+    rdfs:range mus:M4_Key ;
+    rdfs:isDefinedBy <http://data.doremus.org/ontology#> ;
+    rdfs:label "U85 has closely related key"@en , "U85 a pour tonalité étroitement lié"@fr ;
+    rdfs:comment "Scope Note: This class allows to connect two M4 Keys that are mutually closely related from an harmonic point of view, e.g. they are commonly used destinations or transpositions in a modulation."@en ,
+    "Scope Note: Cette classe permet de connecter deux M4 Tonalité qui sont étroitement liées d'un point de vue harmonique, par ex. ces sont des destinations ou des transpositions fréquemment utilisées dans une modulation"@fr ,
```
Comment (rtroncy) [inline doremus.ttl]: Line 1633: s/ces sont/ceux sont

Comment (rtroncy) [reply doremus.ttl]: You may also want to add the 3 wikipedia pages linked from https://github.com/DOREMUS-ANR/knowledge-base/issues/38 in the rdfs:comment as see also examples

**Review by rtroncy — DISMISSED** [review]

File: doremus.ttl (hunk 1 above)
Comment (pasqLisena) [reply doremus.ttl]: Sorry, I am not getting, can you make this change request more explicit please?

File: doremus.ttl (hunk 2 above)
Comment (pasqLisena) [reply doremus.ttl]: Ok for line 1633.

What about `rdfs:seeAlso`?

File: doremus.ttl (hunk 1 above)
Comment (rtroncy) [reply doremus.ttl]: Replace 'par rapport à l'armure' by 'par rapport à l'autre' in the French comment

File: doremus.ttl (hunk 2 above)
Comment (rtroncy) [reply doremus.ttl]: I'm not saying you should add a rdfs:seeAlso property, I'm saying that you can extend your rdfs:comment with more text, and the same way you provide a "scope note" and an example, you can just add "see also https://en.wikipedia.org/wiki/Relative_key" or https://en.wikipedia.org/wiki/Parallel_key or https://en.wikipedia.org/wiki/Closely_related_key

File: doremus.ttl (hunk 1 above)
Comment (pasqLisena) [reply doremus.ttl]: Ah, I wanted to say exactly armure. See [wikipedia](https://fr.wikipedia.org/wiki/Tonalité_relative) : 
> ... qu'une même armure est partagée entre une tonalité majeure et une tonalité mineure

The point: I do not know how to make "relative relationship" in French. I would say "relation relative" but it is not sounds correct. So I preffered to specify that they are relatives in the sense of the key signature ("armure").

Comment (rtroncy) [reply doremus.ttl]: Ah, my bad, or more precisely my ignorance of the proper definition of this concept. Your description is good!

**Review by rtroncy — APPROVED** [review]

**Comment by ccecconi** [comment]
Propriétés validées par le groupe modélisation.

**Pull request merged**